pkg/function/server/server.go

```diff
@@ -27,107 +28,59 @@ func New() *server {
 }
 
 // RegisterMapper registers the map operation handler to the server.
-// Example:
-//
-//	func handle(ctx context.Context, keys []string, data functionsdk.Datum) functionsdk.Messages {
-//		_ = data.EventTime() // Event time is available
-//		_ = data.Watermark() // Watermark is available
-//		return functionsdk.MessagesBuilder().Append(functionsdk.NewMessage(data.Value()))
-//	}
-//
-//	func main() {
-//		server.New().RegisterMapper(functionsdk.MapFunc(handle)).Start(context.Background())
-//	}
```

Comment:
I am removing this comment because it's a duplicate with what we have in doc.go . Would like to maintain a single source of truth regarding how to use map, which is the flat map example. Same for below